frontend/src/metabase-types/store/embed.ts

```diff
@@ -9,6 +11,7 @@ export interface EmbedOptions {
   header?: boolean;
   additional_info?: boolean;
   action_buttons?: boolean;
+  theme?: DisplayTheme;
```

Comment:
I wouldn't put `theme` here. The way we define `EmbedOptions` is a little vague right now. We have 2 types of embedding offering at the moment.
1. Static embedding
2. interactive embedding

And both have their own set of available options. For example, static embedding can set `theme`, but interactive embedding can't, and interactive embedding can set `header=false` while static embedding can't.

The type `EmbedOptions` here is for interactive embedding. I've made a quick search and we seem to use this incorrectly in some places e.g.

https://github.com/metabase/metabase/blob/02afd35b10331b372b35e7aaae8cfc22edd64009/frontend/src/metabase/dashboard/containers/DashboardSharingEmbeddingModal/DashboardSharingEmbeddingModal.tsx#L39-L40

it should be `EmbeddingParameters` instead.

Going forward, I think we should be explicit about options available for Static embedding and interactive embedding.

Maybe we could name them
- `InteractiveEmbeddingOptions`
- `StaticEmbeddingOptions` (if we ever need this type)

But for simplicity, I'd just remove `theme` here.